### ISSUE TITLE
Add configuration option for JaCoCo's includeNoLocationClasses

### ISSUE THREAD

**Issue by Rolf-Smit** [NeoTech-Software/Android-Root-Coverage-Plugin#26]
See #18

**Comment by TonyTangAndroid** [comment]
Thanks for creating this issue. Just to provide another subtle difference that I have noticed,

 for `'com.android.application'` module , it is required to add the extra configuration (You could refer to this pull request to assert this statement https://github.com/TonyTangAndroid/NeoTechCodeCoverageSample/pull/3/files). 

However, for `'com.android.library'`, it is not required. You could refer to this pull request to assert this statement https://github.com/TonyTangAndroid/NeoTechCodeCoverageSample/pull/5/files.

 
```
//Enable RobolectricTestRunner  https://github.com/NeoTech-Software/Android-Root-Coverage-Plugin/issues/18#issuecomment-576630461
tasks.withType(Test) {
    jacoco.includeNoLocationClasses = true
}
```

**Comment by Rolf-Smit** [comment]
This feature has been added and will be released when 1.4 is ready.

**Comment by Rolf-Smit** [comment]
Version 1.4.0 is now available.
